whatapi.py

```diff
@@ -86,10 +86,17 @@ def __init__(self, username=None, password=None, endpoint=None, totp=None):
 
     def _login(self):
         '''Logs in user and gets authkey from server'''
-        loginpage = '{0}/login.php'.format(self.endpoint)
-        data = {'username': self.username,
-                'password': self.password}
-        r = self.session.post(loginpage, data=data)
+        if self.endpoint == "https://orpheus.network":
+            self.session.open('https://orpheus.network/login.php')
+            self.session.select_form()
+            self.session['username'] = self.username
+            self.session['password'] = self.password
+            self.session['keeplogged'] = '1'
+            r = self.session.submit_selected()
+        else:
+            data = {'username': self.username,
```

Comment:
`loginpage` is undefined in this branch.

```suggestion
            loginpage = '{0}/login.php'.format(self.endpoint)
            data = {'username': self.username,
```